sklearn/manifold/_mds.py

```diff
@@ -357,7 +359,8 @@ class MDS(BaseEstimator):
     hypothesis" Kruskal, J. Psychometrika, 29, (1964)
 
     """
-    def __init__(self, n_components=2, metric=True, n_init=4,
+    @_deprecate_positional_args
+    def __init__(self, *, n_components=2, metric=True, n_init=4,
```

Comment:
Not sure if n_components should be positional here and in other manifold embedders (where it is the first arg atm)?

Reply:
'LocallyLinearEmbedding': `n_neighbors` is the first one and `n_components` the second arg, and in one of the tests we were passing positionally the wrong way around. So I thought it seems it's good to have it kwonly.

Reply:
Yes, for LLE is is ambiguous. Maybe the others shouldn't suffer for its faults?